R/stubs.R

```diff
@@ -31,9 +31,12 @@ askForPassword <- function(prompt = "Please enter your password") {
   callFun("askForPassword", prompt)
 }
 
+#' Get Path to Active RStudio Project
 #' @export
 getActiveProject <- function() {
-  callFun("getActiveProject")
+  path <- callFun("getActiveProject")
+  Encoding(path) <- "UTF-8"
```

Comment:
`path` can be `NULL` when there is no active project. Can we guard this with an `is.character()` check?

Reply:
I've added a commit to include a NULL check using `is.null`.